docs/userguide/icu4j/index.md

```diff
@@ -179,7 +257,91 @@ The complete API documentation for ICU4J (javadoc) is available on the ICU4J web
 *   Unicode Text [Compression](https://unicode-org.github.io/icu-docs/apidoc/released/icu4j/com/ibm/icu/text/UnicodeCompressor.html) & [Decompression](https://unicode-org.github.io/icu-docs/apidoc/released/icu4j/com/ibm/icu/text/UnicodeDecompressor.html) – 2:1 compression on English Unicode text.
 *   Collation – [Rule-based sorting](https://unicode-org.github.io/icu-docs/apidoc/released/icu4j/com/ibm/icu/text/RuleBasedCollator.html), [Efficient multi-lingual searching](https://unicode-org.github.io/icu-docs/apidoc/released/icu4j/com/ibm/icu/text/StringSearch.html), [Alphabetic indexing](https://unicode-org.github.io/icu-docs/apidoc/released/icu4j/com/ibm/icu/text/AlphabeticIndex.html)
 
-## How to Install and Build
+## How to Install and Build - ICU 74 and later
+
+Using a pre-built version of ICU from Maven Central can be achieved simply by using the artifact
+coordinates as described above in the Maven portion of the [How to Download ICU4J](#how-to-download-icu4j)
+section.
+
+For non Maven-aware builds, to install ICU4J, simply place the pre-built jar file `icu4j.jar` on your Java `CLASSPATH`.
+If you need Charset API support please also place `icu4j-charset.jar` on your class path along with `icu4j.jar`.
+
+To build ICU4J, you will need a version of Maven and JDK supported by the version of ICU
+and by the Maven build configuration. See [Maven Setup for Java](../../devsetup/java/maven).
+
+Once the JDK and Maven are installed, run the desired Maven target. For example:
+
+~~~
+~/icu/icu4j$ mvn test
```

Comment:
mvn verify
?
Let's not encourage people to skip the integration tests :-)

Reply:
Done.